docs/docs-content/vm-management/vm-migration-assistant/migrate-vms-vmo-cluster.md

```diff
@@ -0,0 +1,929 @@
+  information, refer to
+  [Repave Behaviors and Configurations](../../clusters/cluster-management/node-pool.md#repave-behavior-and-configuration)
+
+  Additionally, we recommend provisioning volumes with the `ReadWriteMany` access mode to ensure that VMs can be
+  [live migrated](https://kubevirt.io/user-guide/compute/live_migration/#limitations).
+
+  ```yaml
+  kubeadmconfig:
+     preKubeadmCommands:
+        # Start containerd with new configuration
+        - systemctl daemon-reload
+        - systemctl restart containerd
+     files:
+        - targetPath: /etc/containerd/config.toml
+        targetOwner: "root:root"
+        targetPermissions: "0644"
+        content: |
+           ## template: jinja
+
+           # Use config version 2 to enable new configuration fields.
+           version = 2
+
+           imports = ["/etc/containerd/conf.d/*.toml"]
+
+           [plugins]
+              [plugins."io.containerd.grpc.v1.cri"]
+              sandbox_image = "registry.k8s.io/pause:3.9"
+              device_ownership_from_security_context = true
+              [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+              runtime_type = "io.containerd.runc.v2"
+              [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+              SystemdCgroup = true
+  ```
+
+  :::
+
+- A VMware vSphere user account with the necessary permissions to manage the VMs you want to migrate.
+  - Migration can be optionally accelerated by providing credentials for the ESXi hosts where the VMs reside.
+- One or more VMs hosted in VMware vSphere. Only VMs whose operating systems are included under
+  [`virt-v2v` supported guest systems](https://libguestfs.org/virt-v2v-support.1.html) can be migrated.
+
+  - If you are migrating more than one VM in the same plan, they must all share the same network.
+  - For cold migrations, ensure that VMs operating Windows are shut down at the virtualized OS level.
+  - For warm migrations,
+    [Changed Block Tracking](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vddk-programming-guide/GUID-7B12E618-7851-4BD3-8E39-819454D8C016.html)
+    must be enabled on your VMs.
+
+<!--prettier-ignore-->
+- The <VersionedLink text="Virtual Machine Migration Assistant" url="/integrations/packs/?pack=vm-migration-assistant-pack"/> pack must be added to your cluster profile. Refer to [Create a VM Migration Assistant Cluster Profile](./create-vm-migration-assistant-profile.md) for guidance.
+  - The VM Migration Assistant service console must be accessible from a web browser.
+
+- We recommend providing a
+  [VMware Virtual Disk Development Kit (VDDK) image](https://developer.broadcom.com/sdks/vmware-virtual-disk-development-kit-vddk/latest)
+  for the migration. This will significantly speed up the migration.
+
+  - The VDDK image must be built and uploaded to your image registry before starting the migration.
```

Comment:
Which registry. This piece with the VDDK images in unclear, even for me. It's required for which component? At source or destination?

Reply:
I've tried making this clearer now.